articles/metadata/lock.md

```diff
@@ -49,7 +49,7 @@ if (id_token) {
 
 You can [update the metadata properties](/metadata/apiv2#update-user-metadata) with calls to the Auth0 Management API.
 
-By including the user's `id_token` in the `Authorization` header, you can make the appropriate `PATCH` call to the [Update a user](/api/management/v2#!/Users/patch_users_by_id) endpoint. 
+By including the user's `id_token` in the `Authorization` header, you can make the appropriate `PATCH` call to the [Update a user](/api/management/v2#!/Users/patch_users_by_id) endpoint.

```

Comment:
I believe this will be removed in the new pipeline, making a note to confirm this.